index.js

```diff
@@ -55,8 +45,24 @@ const updateMap = (mapToDisplay) => {
       const vectorLayer = new VectorLayer({
         source: new VectorSource({
           features: (new GeoJSON()).readFeatures(data)
-        }),
-        style: styleFunction
+        })
+      });
+
+      vectorLayer.getSource().forEachFeature(function(feature) {
+        styles = {
+          'Polygon': new Style({
+            stroke: new Stroke({
+              color: 'black',
+              lineDash: [4],
+              width: 3
+            }),
+            fill: new Fill({
+              color: 'rgba(' + Math.floor(Math.random() * 256) + ", " + Math.floor(Math.random() * 256)
+              + ', ' + Math.floor(Math.random() * 256) + ', 0.3)'       
```

Comment:
This is a great start, but we'd like a better "graph coloring" solution. I'll forward you some stuff.